cms/templates/index.html

```diff
@@ -24,13 +24,13 @@ <h3 class="sr">${_("Page Actions")}</h3>
           % if course_creator_status=='granted':
           <a href="#" class="button new-button new-course-button"><i class="icon fa fa-plus icon-inline"></i>
               ${_("New Course")}</a>
-            % if libraries_enabled:
-              <a href="#" class="button new-button new-library-button"><i class="icon fa fa-plus icon-inline"></i>
-              ${_("New Library")}</a>
-            % endif
           % elif course_creator_status=='disallowed_for_this_site' and settings.FEATURES.get('STUDIO_REQUEST_EMAIL',''):
           <a href="mailto:${settings.FEATURES.get('STUDIO_REQUEST_EMAIL','')}">${_("Email staff to create course")}</a>
           % endif
+          % if libraries_enabled:
```

Comment:
I'd prefer that our UI logic be more specific to the experience/behavior we expect.  So, rather than saying 'render the New Library' button by checking to see if libraries_enabled = True, we should instead encapsulate the rules into something like a 'show_new_library_button' operation.  This way as rules are added/modified the UI code will not need to be updated.